src/gui/qgsmapcanvasmap.h

```diff
@@ -16,22 +16,18 @@
 #ifndef QGSMAPCANVASMAP_H
 #define QGSMAPCANVASMAP_H
 
-#include <QGraphicsRectItem>
-#include <QPixmap>
-#include <QTimer>
-
-#include <qgis.h>
-#include <qgsmaptopixel.h>
-
 #include <qgsmapcanvasitem.h>
 
 class QgsMapSettings;
 class QgsMapCanvas;
 
 /** \ingroup gui
  * A rectangular graphics item representing the map on the canvas.
+ *
+ * @note This class is not a part of public API
+ * @note not available in Python bindings
```

Comment:
Yep - marking them private will exclude them from the sip/doc coverage test too. I'd still leave the existing docs in place though, since they are useful for QGIS devs.